.circleci/config.yml.disabled

```diff
@@ -113,50 +113,6 @@ commands:
           path: acceptance-tests/tests/build/jvmErrorLogs
 
 jobs:
```

Comment:
Rather than deleting parts of it and renaming it we should just delete the whole file.

Reply:
That can be arranged :)